design-toolkit/components/src/components/skeleton/index.tsx

```diff
@@ -49,6 +49,10 @@ export function Skeleton({
   shape = SkeletonStylesDefaults.shape,
   ...rest
 }: SkeletonProps) {
-  return <div {...rest} className={SkeletonStyles({ className, shape })} />;
```

Comment:
If you remove the `children` from the destructured props on line 47, then it'll automatically spread along with `rest`